Products/CMFPlomino/fields/datetime.py

```diff
@@ -148,6 +159,11 @@ def processInput(self, submittedValue):
                 return StringToDate(submittedValue, fmt)
         # it is instance type when no js is detected
         # submittedValue = ampm: , day: 09, hour: 00, minute: 00, month: 03, year: 1993
+        if submittedValue.year == '0000' or \
+           submittedValue.month == '00' or \
+           submittedValue.day == '00':
+            return None
+
```

Comment:
I agree: if the user just forgot to enter one of the value, yes we should raise a validation issue, and not assume it is equivalent to an empty value.
At the contrary if all the values are null, it means the user did not enter anything.